jest.config.sdk.js

```diff
@@ -8,9 +8,9 @@ module.exports = {
     coveragePathIgnorePatterns: ['/node_modules/', '/test/'],
     coverageThreshold: {
         './src/': {
-            branches: 87,
-            functions: 88,
-            lines: 96,
```

Comment:
We shouldn't decrease the coverage here. Can you identify and add enough tests to make the coverage?